### PR TITLE
97064-update-style-guide-to-use-more-semantic-classes

### PR DESCRIPTION
**Pull request by joebri**
97064
### Review
 - [x] @jtrinklein

### After Merge

- [ ] [site](https://daptiv.github.io/style-guide) is updated

### CODE REVIEW

File: styles/nav.scss [hunk 1]
```diff
@@ -119,7 +119,7 @@ $daptiv-subnav-tab-h-pad: 14px;
         }
     }
 
-    .daptiv-nav-menuitem-alternate {
+    .daptiv-nav-menuitem.alternate {
```
Comment (jtrinklein) [inline styles/nav.scss]: this should be nested under the .daptiv-nav-menuitem rule

Comment (joebri) [inline styles/nav.scss]: The SCSS linter complains at any nesting greater than 3 levels.

Comment (jtrinklein) [inline styles/nav.scss]: hmmmm.... that's unfortunate. Well, if we're going to leave it separate, we should at least move it up near the other rule.  Also this rule doesn't need to `@extend .daptiv-nav-menuitem-base`

Comment (joebri) [inline styles/nav.scss]: The only construct between the 'alternate' rule and daptiv-nav-menuitem is the 'disabled' rule on the same class. Do you have some guidelines on ordering of these 'decorator' rules? 
Have removed the @extend rule.

Comment (jtrinklein) [inline styles/nav.scss]: ah sorry, i only read half of the previous rule, thought it was subnav only. Position is :+1:

Comment (joebri) [inline styles/nav.scss]: Ah - okay. Tak!

**Comment by joebri** [comment]
@jtrinklein Do you want to take another quick look? Thanks.

**Comment by jtrinklein** [comment]
:tophat:
:godmode:
:shirt::+1:
:jeans:

**Pull request merged**